symbology/simple-marker-symbol/src/main/java/com/esri/samples/simple_marker_symbol/SimpleMarkerSymbolSample.java

```diff
@@ -51,8 +52,12 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create ArcGISMap with imagery basemap
-      final ArcGISMap map = new ArcGISMap(Basemap.createImagery());
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create ArcGISMap with imagery basemap style
+      final ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_IMAGERY_STANDARD);
 
       // create spatial reference for WGS 1948
```

Comment:
If you're feeling up for it, you could clean up lines 62-67 as well to match the previous samples in this review?

Reply:
Tidied up